deploy/compose/docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-
+version: "3.9"
```

Comment:
This is deprecated from the docker-compose file.

Reply:
Ok, removed